### PR TITLE
Tg feature campaignmodal

### PR DESCRIPTION
**Pull request by talgrewal**
# Tg feature campaignmodal

- Add Content and style to Campaign modal 
- Add logic to open and close modal
- Add ability to subscribe to campaigns from campaign modal

### CODE REVIEW

File: client/js/screens/CampaignModal/CampaignModal.js [hunk 1]
```diff
@@ -1,8 +1,92 @@
 import React from 'react';
-import {Text} from 'react-native';
+import {Text, View, ScrollView, Image, Linking} from 'react-native';
+import {withNavigation} from 'react-navigation';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+import styles from '../Campaign/styles';
+import {colors} from '../../config/styles';
+// import SafeAreaView from 'react-native-safe-area-view';
```
Comment (siddharthparmar7) [inline client/js/screens/CampaignModal/CampaignModal.js]: Please remove commented code if not needed.

File: client/js/screens/CampaignModal/CampaignModal.js [hunk 2]
```diff
@@ -1,8 +1,92 @@
...
+import PropTypes from 'prop-types';
+import BackButton from '../../assets/miscicons/xicon.png';
+import MainSubscribeButton from '../../components/MainSubscribeButton';
+import MainUnsubscribeButton from '../../components/MainUnsubscribeButton';
 
-const CampaignModal = props => {
-  return <Text>Event More Screen</Text>;
+const CampaignModal = ({navigation, campaignData, user}) => {
+  console.log(user);
```
Comment (siddharthparmar7) [inline client/js/screens/CampaignModal/CampaignModal.js]: remove this console log.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please explain why did you delete client/js/screens/CampaignModal/styles.js file.

**Comment by siddharthparmar7** [comment]
And remove .DS_Store from your branch. Update your .gitignore file to not upload it again.

**Comment by talgrewal** [comment]
Writing this in response to your comment above: 
Please explain why did you delete client/js/screens/CampaignModal/styles.js file.

I removed the file because I realized there is tons of css that is duplicate of the campaign page. Tien suggested this on another page but I largly reused css from the campaign page so I deleted the styles files and linked to the other file.

File: client/js/screens/CampaignModal/CampaignModal.js (hunk 2 above)
Comment (talgrewal) [reply client/js/screens/CampaignModal/CampaignModal.js]: Done

File: client/js/screens/CampaignModal/CampaignModal.js (hunk 1 above)
Comment (talgrewal) [reply client/js/screens/CampaignModal/CampaignModal.js]: Done

**Comment by talgrewal** [comment]
Ready for review.

**Comment by siddharthparmar7** [comment]
Nice work. Please resolve merge conflicts.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**